tests/test_integrations/test_crowdstrike.py

```
@@ -1,17 +1,71 @@
+def executor(context):
+    identity_protection = context["clients"]["identity_protection"]
+    result = []
+    try:
+        response = identity_protection.query_sensors(limit=10)
+        result.append(response)
+    except Exception as e:
+        result.append({"error": str(e)})
+    return [{"result": result}]
+"""
+
+def _get_tokens(get_keys):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_get_tokens`

(ANN202)